CONTRIBUTING

```diff
@@ -0,0 +1,31 @@
+# CONTRIBUTING
+
+When contributing code to the repository, here are a few things to know:
+
+## CODE
+
+- Push feature branches to your own fork and make pull requests from there to Mozilla.
+- The pull-requests will all run in CircleCI.
+- Every new developer feature should come with documentation. E.g. "How to run lint checking". New tools and tricks that comes in without documentation is a fail.
+- Avoid global CSS that isn't tied to a specific component.
+- Run lint check and all tests locally before making a pull request using `yarn run lint`. In any case, pre-commit hooks will block your commit if there are linting issues. You can (but should not) skip the pre-commit hooks with `git commit --no-verify ...`.
```

Comment:
```md
- Run lint check and all tests locally before making a pull request using `yarn run lint`. 
  - This will be enforced by pre-commit hooks that run the linter before every commit. You can (but should not) skip the pre-commit hooks with `git commit --no-verify ...`.
```